lib/postgresql.dart

```diff
@@ -157,6 +157,23 @@ abstract class Row {
   
   /// An unmodifiable map of column names and values. 
   Map toMap();
+
+  List<Column> toColumns();
+}
+
+@proxy
```

Comment:
I wonder why a proxy annotation is used here? _Column  doesn't seems to use noSuchMethod.
A copy and paste mistake?